frontend/src/components/DataCard/DataCardAdjudicateButton.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Fragment, useState } from "react";
```

Comment:
Would it make sense to split this folder out or give it some sub-folders? The list of files is just getting pretty large.